server/api/products.js

```diff
@@ -48,7 +48,7 @@ router.post('/', async (req, res, next) => {
 router.put('/productId', async (req, res, next) => {
   try {
     const {productId} = req.params
-    const product = await Product.findByPk(productId)
+    let product = await Product.findByPk(productId)
 
     product = await product.update(req.body)
 
```

Comment:
Line 51 was changed because commit linter complained about assigning to a constant on line 53. The other changes are Prettier re-formatting.